src/tezos/michelson.ml

```diff
@@ -7,4 +7,10 @@ type t = Michelson_v1_primitives.prim Tezos_micheline.Micheline.canonical
 let expr_encoding =
   Micheline_encoding.canonical_encoding ~variant:"michelson_v1"
     Michelson_v1_primitives.prim_encoding
-let lazy_expr_encoding = Data_encoding.lazy_encoding expr_encoding
+
+let unit =
+  Micheline.strip_locations (Prim (-1, Michelson_v1_primitives.D_Unit, [], []))
+let is_unit value =
```

Comment:
Is there a point to define this function globally while it is used only once in `operation:encoding`?

Reply:
Important thing, it's not "globally", it's inside of a module, which is inside of a library, while it is accessible globally it's not like it's polluting the scope. It's quite different from Haskell in this sense.

But this was on the other PR and the reasons why I added on Michelson instead of just doing it inline, is because I was following a similar style from Tezos and thinking now(after the fact, which is biased), it makes sense conceptually, I would prefer to avoid comparing Michelson outside of this module in general.

Reply:
Didn't see it was inside a module. Thanks.